app/services/distribution_update_service.rb

```diff
@@ -3,15 +3,16 @@ class DistributionUpdateService
 
   def initialize(old_distribution, new_distribution_params)
     @distribution = old_distribution
-    @params = new_distribution_params
+    @old_issued_at = old_distribution.issued_at
+    @params = new_distribution_params.with_indifferent_access
```

Comment:
```suggestion
    @params = new_distribution_params
    unless @params.is_a?(ActionController::Parameters)
         @params = @params.with_different_access
    end
```
Based on the broken specs and memory (if it serves me right), the `ActionController::Parameters` already has indifferent access.

Reply:
We should be able to safely apply `with_indifferent_access` even if it's already had it applied -- I don't think we need to type-check it first. We can assertively tell it to behave like we want it to :muscle: